docs/contributing/contributing.md

```
@@ -0,0 +1,173 @@
+pixi run pre-commit
+
+# Run on all files
+pixi run pre-commit-all
+```
+
+## Building Documentation
+
+The documentation site is built with [MkDocs](https://www.mkdocs.org/) and the
+[Material](https://squidfunk.github.io/mkdocs-material/) theme.
+
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)